jquery.flot.axislabels.js

```diff
@@ -100,10 +100,10 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
                         var x, y;
                         if (axisName.charAt(0) == 'x') {
                             x = plot.getPlotOffset().left + plot.width()/2 - width/2;
-                            y = plot.getCanvas().height;
+                            y = plot.getCanvas().height - 3;
```

Comment:
random number?

Reply:
Of course, a random number is as valid as any other :trollface: